knowledge-base/disable-gesture-open-sidedrawer-net-maui-winui.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Disabling Gesture to Open Drawer in .NET MAUI SideDrawer for WinUI
+description: Learn how to configure the RadSideDrawer in .NET MAUI so it can only be opened through a button click on WinUI, disabling the gesture control.
+type: how-to
+page_title: How to Disable Gesture Opening of RadSideDrawer in .NET MAUI on WinUI
+slug: disable-gesture-open-sidedrawer-net-maui-winui
+tags: sidedrawer, .net maui, winui, gesture, open, button
+res_type: kb
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 7.1.0 | Telerik UI for .NET MAUI SideDrawer| [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova)| 
+
+## Description
+
+In a .NET MAUI application, you might want to restrict opening of the [SideDrawer]({%slug sidedrawer-overview%}) to occur only through a button click, especially on the WinUI platform, instead of using the swipe/drag gesture. This can be essential for cases where gesture control needs to be disabled for specific UI requirements.
```

Comment:
```suggestion
In a .NET MAUI application, you might want to restrict the [SideDrawer]({%slug sidedrawer-overview%}) to open only through a button click, especially on the WinUI platform, instead of using the swipe/drag gesture. This can be essential for cases where gesture control needs to be disabled for specific UI requirements.
```